migrations.json

```diff
@@ -0,0 +1,108 @@
+{
+  "migrations": [
```

Comment:
What is this file for? Is it related to schematics, or to nx?

Reply:
It's auto generated by Nx.